.github/workflows/main.yml

```diff
@@ -44,6 +46,12 @@ jobs:
         run: |
           pip install --upgrade cloudpickle pytest pytest-cov responses
           make unit-tests
+      - name: test source distribution
+        if: matrix.task == 'sdist'
+        shell: bash
+        run: |
+          python setup.py sdist
+          pip install dist/dask-pytorch-$(cat VERSION).tar.gz
```

Comment:
I ran this once before including any fixes. Can confirm that it reproduces the problem reported in #4.

logs: https://github.com/saturncloud/dask-pytorch/pull/7/checks?check_run_id=1421317551

```text
distribution option: 'test_require'
  warnings.warn(msg)
Processing ./dist/dask-pytorch-0.1.0.tar.gz
    ERROR: Command errored out with exit status 1:
     command: /usr/share/miniconda/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-uaznqvv8/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-uaznqvv8/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-lo06y06o
         cwd: /tmp/pip-req-build-uaznqvv8/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-uaznqvv8/setup.py", line 7, in <module>
        with open("VERSION", "r") as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'VERSION'
```